### PR TITLE
interface: only display our contact when appropriate

### PR DESCRIPTION
**Pull request by tacryt-socryp**
Fix https://github.com/urbit/landscape/issues/572

### CODE REVIEW

File: pkg/interface/src/views/apps/chat/components/ChatWindow.tsx [hunk 1]
```diff
@@ -317,7 +323,7 @@ class ChatWindow extends Component<
           onScroll={this.onScroll}
           data={graph}
           size={graph.size}
-          pendingSize={pendingSize}
+          pendingSize={pendingSize + contactsModified}
```
Comment (matildepark) [inline pkg/interface/src/views/apps/chat/components/ChatWindow.tsx:326]: Why are we adding 100 if we hide our contact?

**Review by matildepark — COMMENTED** [review]
Mostly what I expect.

File: pkg/interface/src/views/apps/chat/components/ChatWindow.tsx (hunk 1 above)
Comment (tacryt-socryp) [reply pkg/interface/src/views/apps/chat/components/ChatWindow.tsx:326]: To force a re-render of the virtual scroller

**Review by matildepark — APPROVED** [review]
We only seem to use pendingSize for triggering rerenders and not for any render-specific logic so I guess this is okay (albeit obscure and a little hacky). I'd put a comment in here to flag this

**Comment by tacryt-socryp** [comment]
Thanks, added a comment.

**Pull request merged**